YAFC/Widgets/ObjectTooltip.cs

```diff
@@ -23,18 +23,18 @@ private void BuildHeader(ImGui gui)
                 if (extendHeader && !(target is Goods))
                     name = name +" (" + target.target.type + ")";
                 gui.BuildText(name, Font.header, true);
-                var milestoneMask = Milestones.Instance.milestoneResult[target.target];
-                if (milestoneMask > 1)
+                var milestoneMask = Milestones.Instance.GetMilestoneResult(target.target);
```

Comment:
I think `var` is good when the type is obvious to a reader. Here, it's not obvious -- one needs to navigate to `GetMilestoneResult` to see what it returns. I think an explicit type would be easier to read. What do you think?

Reply:
I tried to keep the same syntax as the rest of the code and it seemed to be using `var` all over the place.

TBH I do not know what best practices are here a well... But I do have some points:
* IDE will open up IntelliSense to give available functions on types.
* When hovering on the `GetMilestoneResult()` function call or the variable itself, it shows that it is of type `Bits`
* `milestoneMask` is a 'milestone mask'... The actual type being `Bits` is (kind of) irrelevant?

Also,
> type is obvious to a reader

is kind of subjective. For me it was obvious and you you it was not...

Maybe we should set some coding style rules to clear these kind of things up a little (and/or the formatter of #3 )?

Reply:
Again, either way is fine for me